salt/reboot/init.sls

```diff
@@ -0,0 +1,42 @@
+##################################################
+# Configuration for the reboot manager
+##################################################
+
+# `max_holders` contains the maximum number of lock holders for the cluster.
+{% set max_holders = salt['pillar.get']('etcd:masters', '1')|int %}
```

Comment:
By the way, since this is a `sls` file you can do `pillar['etcd']['masters']` directly.